packages/allow-scripts/src/config.js

```diff
@@ -10,6 +10,7 @@ const normalizeBin = require('npm-normalize-package-bin')
 const { loadCanonicalNameMap } = require('@lavamoat/aa')
 
 const bannedBins = new Set(['corepack', 'node', 'npm', 'pnpm', 'yarn'])
+const DEFAULT_LIFECYCLE_EVENTS = ['preinstall', 'install', 'postinstall']
```

Comment:
pls add description. how do we use these?

Reply:
Added oneline description.